beacon_node/network/src/sync/block_lookups/single_block_lookup.rs

```diff
@@ -110,19 +110,18 @@ impl<L: Lookup, T: BeaconChainTypes> SingleBlockLookup<L, T> {
         &mut self,
         cx: &SyncNetworkContext<T>,
     ) -> Result<(), LookupRequestError> {
-        let block_root = self.block_root();
         let block_already_downloaded = self.block_already_downloaded();
         let blobs_already_downloaded = self.blobs_already_downloaded();
 
-        if block_already_downloaded && blobs_already_downloaded {
-            trace!(cx.log, "Lookup request already completed"; "block_root"=> ?block_root);
```

Comment:
Is the deletion of this trace log statement intentional?

Reply:
yeah.. I don't think this log is very useful. There are hardly any tracing logs in this code & I just spent a long time tracking down a bug in this code & it required me to put a bunch of additional logs all over the place. I never turned on trace logs because there are hardly any of them and none of them would've helped me debug anyway, they would've only cluttered the output and made it more difficult.